core/audits/network-requests.js

```diff
@@ -77,6 +83,7 @@ class NetworkRequests extends Audit {
         priority: record.priority,
         isLinkPreload,
         experimentalFromMainFrame,
+        entityName,
```

Comment:
We could name this as `entity` to match the pattern of audit-side classification. [Third-party summary audit](https://github.com/GoogleChrome/lighthouse/blob/main/core/audits/third-party-summary.js#L220) is a good example. 

When the `entity` property is provided from an audit, the report rendering side skips classification and relies on the provided one. See [report-side classification](https://github.com/GoogleChrome/lighthouse/blob/main/report/renderer/report-utils.js#L124) yielding to audit provided `entity`. 

In this case the audit isn't rendered, but would be good to keep the same convention.

Reply:
Thanks for review @alexnj. PTAL 🙏